facade/logstash.go

```diff
@@ -180,9 +183,34 @@ func getFilters(services []servicedefinition.ServiceDefinition, filterDefs map[s
 	return filters
 }
 
+func getAuditLogSection(services []servicedefinition.ServiceDefinition, auditTypes *[]string) string {
```

Comment:
Can you write some unit-test for this?  I can point you at some examples

The tests should cover several cases:
- no services
- a list of services, but no audit files
- a list of services with multiple audit files for the same audit type
- a list of services with multiple audit files for different audit types